Parking-Lot/src/com/practice/parkinglot/Driver.java

```diff
@@ -0,0 +1,159 @@
+		printFreeSlots(orionMallParkingLot, new Truck());
+		printOccupiedSlots(orionMallParkingLot, new Car());
+		printOccupiedSlots(orionMallParkingLot, new Bike());
+		printOccupiedSlots(orionMallParkingLot, new Truck());
+		
+		Ticket ticket1 = parkVehicle(new Car("KA-01-DB-1541", "black"), "KA-01-DB-1541", "Black", orionMallParkingLot);
+		Ticket ticket2 = parkVehicle(new Car("KA-02-CB-1334", "red"), "KA-02-CB-1334", "red", orionMallParkingLot);
+		Ticket ticket3 = parkVehicle(new Car("KA-01-DB-1133", "Black"), "KA-01-DB-1133", "Black", orionMallParkingLot);
+		Ticket ticket4 = parkVehicle(new Car("KA-05-HJ-8432", "white"), "KA-05-HJ-8432", "white", orionMallParkingLot);
+		Ticket ticket5 = parkVehicle(new Car("WB-45-HO-9032", "white"), "WB-45-HO-9032", "white", orionMallParkingLot);
+		Ticket ticket6 = parkVehicle(new Car("KA-01-DF-8230", "Black"), "KA-01-DF-8230", "Black", orionMallParkingLot);
+		Ticket ticket7 = parkVehicle(new Car("KA-21-HS-2347", "red"), "KA-21-HS-2347", "red", orionMallParkingLot);
+		
+		countFreeSlotsCar(orionMallParkingLot);
+		countFreeSlotsBike(orionMallParkingLot);
+		countFreeSlotsTruck(orionMallParkingLot);
+		
+		unparkVehicle(ticket1, orionMallParkingLot);
+		unparkVehicle(ticket1, orionMallParkingLot);
+		unparkVehicle(ticket1, orionMallParkingLot);
+		/*unparkVehicle(ticket4, orionMallParkingLot);
+		unparkVehicle(ticket5, orionMallParkingLot);
+		unparkVehicle(ticket6, orionMallParkingLot);
+		unparkVehicle(ticket7, orionMallParkingLot);*/
+		
+		countFreeSlotsCar(orionMallParkingLot);
+		countFreeSlotsBike(orionMallParkingLot);
+		countFreeSlotsTruck(orionMallParkingLot);
+		printFreeSlots(orionMallParkingLot, new Car());
+		printFreeSlots(orionMallParkingLot, new Bike());
+		printFreeSlots(orionMallParkingLot, new Truck());
+		printOccupiedSlots(orionMallParkingLot, new Car());
+		printOccupiedSlots(orionMallParkingLot, new Bike());
+		printOccupiedSlots(orionMallParkingLot, new Truck());
+		
+		Ticket ticket8 = parkVehicle(new Bike("KA-01-DB-1541", "black"), "KA-01-DB-1541", "black", orionMallParkingLot);
+		Ticket ticket9 = parkVehicle(new Truck("KA-32-SJ-5389", "orange"), "KA-32-SJ-5389", "orange", orionMallParkingLot);
+		Ticket ticket10 = parkVehicle(new Truck("KL-54-DN-4582", "green"), "KL-54-DN-4582", "green", orionMallParkingLot);
+		Ticket ticket11 = parkVehicle(new Truck("KL-12-HF-4542", "green"), "KL-12-HF-4542", "green", orionMallParkingLot);
+		
+		countFreeSlotsCar(orionMallParkingLot);
+		countFreeSlotsBike(orionMallParkingLot);
+		countFreeSlotsTruck(orionMallParkingLot);
+		printFreeSlots(orionMallParkingLot, new Car());
+		printFreeSlots(orionMallParkingLot, new Bike());
+		printFreeSlots(orionMallParkingLot, new Truck());
+		printOccupiedSlots(orionMallParkingLot, new Car());
+		printOccupiedSlots(orionMallParkingLot, new Bike());
+		printOccupiedSlots(orionMallParkingLot, new Truck());
+	}
+	public static void unparkVehicle(Ticket ticket, ParkingLot orionMallParkingLot) {

```

Comment:
Apart from the main method here, every other method should not have been part of the Driver class.